app/views/helpers/FullResultsPageHelper.scala

```diff
@@ -124,10 +124,62 @@ object FullResultsPageHelper extends ViewHelper {
         case (y1: FlatRate, y2: MarginalRate)     => nonTabDisplay(Seq(y1, y2))
         case _                                    => throw new RuntimeException("Both financial years are flat rate")
       }
+    }
+
+    val financialYearTables = calculatorResult.fold(single => nonTabDisplay(Seq(single.details)))(dualResultTable)
+
+    Html(
+      Seq(
+        financialYearTables.body,
+        whatIsMarginalRate(calculatorResult)
+      ).mkString
+    )
+
+  }
+
+  private def marginalReliefFormula(implicit messages: Messages): String =
+    s"""<h3 class="govuk-heading-s" style="margin-bottom: 4px;">${messages(
+        "fullResultsPage.marginalReliefFormula"
+      )}</h3>
+       |<p class="govuk-body">${messages("fullResultsPage.marginalReliefFormula.description")}</p>""".stripMargin
 
+  private def whatIsMarginalRate(calculatorResult: CalculatorResult)(implicit messages: Messages) = {
+
+    val show = {
+      val taxDetails = calculatorResult.fold(single => Seq(single.details))(dual => Seq(dual.year1, dual.year2))
+      taxDetails.exists(taxDetails => taxDetails.fold(_ => false)(isFiveStepMarginalRate))
     }
+
+    if (show) {
+      Html(
+        Seq(
+          marginalReliefFormula,
+          govukDetails(
+            Details(
+              summary = Text(messages("fullResultsPage.whatIsMarginalRateFraction")),
+              content = HtmlContent(
+                s"""<p>${messages("fullResultsPage.details.standardFraction")}</p>
+                   |    <p>${messages("fullResultsPage.details.standardFractionExample")}</p>
+                   |    <p><b>${messages("fullResultsPage.details.whatIsMarginalRate")}</b></p>
+                   |    <p>${messages("fullResultsPage.details.smallProfitRate")}</p>
+                   |    <p>
+                   |        ${messages("fullResultsPage.details.examples.1")}<br/>
+                   |        ${messages("fullResultsPage.details.examples.2")}<br/>
+                   |        ${messages("fullResultsPage.details.examples.3")}<br/>
+                   |        ${messages("fullResultsPage.details.examples.4")}<br/>
+                   |    </p>""".stripMargin
+              )
+            )
+          ).body,
+          hr.body
+        ).mkString
+      )
+    } else { Html("") }
```

Comment:
We have HtmlFormat.empty for this. Also, instead of alway returning an Html, you could return Option[Html]